installing/installing_aws/installing-aws-localzone.adoc

```diff
@@ -6,16 +6,20 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-In {product-title} version {product-version}, you can install a cluster on Amazon Web Services (AWS) into an existing VPC, extending workers to the edge of the Cloud Infrastructure using AWS Local Zones.
+In {product-title} version {product-version}, you can install a cluster on Amazon Web Services (AWS) into an existing VPC, extending workers to the edge of the Cloud Infrastructure by using AWS Local Zones.
```

Comment:
On your PR, I see https://github.com/openshift/installer/pull/7137/files#diff-d4da2968a241c2898ad36d96733640d0392692a1c7c1941a0a80e120a4f0af32R29-R69:~:text=Auto%20opt%2Din%20Zone%20Groups%3A%20the%20zone%20groups%20from%20the%20Local%20Zone%20added%20to%20the%20install%2Dconfig.yaml%20will%20automatically%20opt%2Din%20before%20the%20subnets%20is%20created. 

Is this the default behaviour? If so, can a user opt out and create their own subnets?

Reply:
The user can choose two ways:
- Install a cluster quickly extending compute nodes to AWS Local Zone (4.14+)  <-- goal of this PR
- Install a cluster in an existing VPC with Local Zone subnets (installer 4.13+)